common/display/src/main/scala/uk/ac/wellcome/display/models/DisplayImageSource.scala

```diff
@@ -24,6 +24,8 @@ object DisplayImageSource {
     imageSource: ImageSource[DataState.Identified]): DisplayImageSource =
     new DisplayImageSource(
       id = imageSource.id.canonicalId,
-      ontologyType = imageSource.ontologyType
+      ontologyType = imageSource match {
+        case SourceWorks(work, _) => DisplayWorkType(work.data.workType)
```

Comment:
What if the canonical and redirected works have different `workType`s? `Work` is never going to be wrong (as the other types are regarded as subclasses of `Work`) so I think it might be more correct to say that an `imageSource` of type `SourceWorks` always has `ontologyType = "Work"`

Reply:
The problem is that the `type` field of the work with that ID is not necessarily going to be `Work`, so thought it would make more sense to match that. I am not sure inheritance really applies here personally, but unsure about this whole ontology business in general so can just return `Work` at all times if you think that makes sense.

Reply:
> the type field of the work with that ID is not necessarily going to be Work

That's true, but remember there are 2 works in `SourceWorks` that are not necessarily going to have the same `type` field - so I think we're forced to return `Work` at all times. The inheritance thing was a biiiiig discussion last week that I suspect we'll want to return to at some point, but for now we are claiming (although not enforcing in code) that these types inherit as I described.

Reply:
Cool makes sense, will change to `Work` :+1: 

Any chance for a very brief summary of the discussion?

Reply:
Some context here: https://github.com/wellcomecollection/catalogue/pull/878#discussion_r492741671

We didn't want to expose `Standard` (as a `type`) in the API and also didn't want to have a `WorkType.Work` given that all `Work`s are of type `Work` (🤯 ). Therefore we say that you can have works of `type` `Collection`, `Series`, or `Section`, all of which are subsets of `Work`. I don't think any of us are totally happy with this (I know I'm not) but it seemed to be the most pragmatic solution for the situation we were/are in.

Reply:
:+1: makes sense.

Maybe we could remove `WorkType.Standard` altogether and make `workType` an optional field? (potentially would make sense to rename it in that case, `WorkSpecialisation` or something..)

Reply:
I think this is where the inheritance stuff falls down because it _is_ useful to know that something is a `WorkType.Standard`, so it doesn't feel like a good idea to make it optional - there's tension between the API/Display model and the internal model

Reply:
I don't see the difference between having it as `None` vs having a separate case class for it. To me it is clearer with the optional field as it's existence suggests it is a more specialised thing, rather than a completely different type.